src/aspire/volume/symmetry_groups.py

```diff
@@ -88,7 +88,7 @@ def parse(symmetry, dtype):
         }
         if symmetry_type not in map_to_sym_group.keys():
             raise ValueError(
-                f"Symmetry type {symmetry_type} not supported. Try: {*map_to_sym_group.keys(),}."
+                f"Symmetry type {symmetry_type} not supported. Try: {*map_to_sym_group.keys(), }."
```

Comment:
Huh? This is just a formatting thing, right?

Reply:
yes, one of the linters was complaining.  I think I was upstream from what we're testing.